en/docs/tutorials/configure-automatic-deployment.md

```diff
@@ -0,0 +1,105 @@
+# Configure Automatic Deployment
+
+When you develop a Choreo component, you need to commit its source code into a GitHub repository. You can select a repository that Choreo manages or a repository that you own and manage.
+
+Choreo-managed repositories are not accessible to external users. Therefore, when multiple developers need to develop components in collaboration, they should connect their own repositories to save the source code. In such organizations, deploying a component manually each time a developer updates the component design can be cumbersome. Therefore, Choreo allows you to configure automatic deployment for your components. When you enable automatic deployment, every commit that changes the component implementation automatically deploys the component to the development environment and triggers a build.
+
+This tutorial walks you through the steps to enable automatic deployment for a component. In this tutorial, you will:
+
+- Configure automatic deployment for a REST API.
+- Commit updates to the REST API implementation and observe automatic builds.
+
+## Step 1: Connect your own GitHub repository for a REST API
+
+Let's first create a REST API connecting a GitHub repository you own to save the implementation.
+
+For instructions, see [Connect Your Own GitHub Repository to Choreo](connect-your-own-github-repository-to-choreo.md)
+
+## Step 2: Enable automatic deployment
+
+To enable automatic deployment for your REST API, follow the steps given below:
+
+1. First, run a manual build. To do this, click **Deploy** in the left pane and then click **Build and Deploy**.
+
+    !!! info
+        Choreo requires you to perform the first deployment of each component manually so that you can provide values for any configurable variables that the implementation may include.
+
+2. Click the **Automatic Deployment** toggle to enable automatic deployment.
+
+    !!! info
+        You need to verify your GitHub account when you enable automatic deployment for the first time.
+
+3. In the message that appears on the bottom-right of the page to configure an alert for build failures, click **Configure Alert**.
```

Comment:
Shall we rephrase this as "In the message that appears on the bottom-right of the page , click **Configure Alert** to configure an alert for build failures."